cli/running/watchdog.go

```diff
@@ -73,16 +71,11 @@ func (wd *Watchdog) Start() {
 		}
 		wd.logger = wd.Instance.logger
 		// Start the Instance and forwarding signals (except  SIGINT and SIGTERM)
-		wd.startSignalHandling()
-		wd.stopMutex.Lock()
-		if !wd.stopped {
-			if err := wd.Instance.Start(); err != nil {
-				wd.logger.Printf(`Watchdog(ERROR): "%v".`, err)
-				wd.stopMutex.Unlock()
-				break
-			}
+		if err := wd.Instance.Start(); err != nil {
+			wd.logger.Printf(`Watchdog(ERROR): "%v".`, err)
+			break
 		}
-		wd.stopMutex.Unlock()
```

Comment:
I think, we have a leak here of the context created in caller of this function. Stop is not called until tt process end. But the context is not used.

Reply:
But we call the stop function in any case, why is it a leak?

Reply:
Replaced using contexts with channels.

Reply:
> But we call the stop function in any case, why is it a leak?

Looks like, we call stop only after Tarantool instance is stopped, after wd.Start exit. Why do we need the context alive while Tarantool is running?